runtime/util/vmargs.c

```diff
@@ -1583,47 +1529,6 @@ addLauncherArgs(J9PortLibrary * portLib, JavaVMInitArgs *launcherArgs, UDATA lau
 			return -1;
 		}
 		optArg->vmOpt.extraInfo = currentOpt->extraInfo;
-
-		if (0 == strncmp(optString, VMOPT_XXMAXDIRECTMEMORYSIZEEQUALS, strlen(VMOPT_XXMAXDIRECTMEMORYSIZEEQUALS))) {
-			/*
-			 *  CMVC 149604 - check for errors in <size> portion of -XX:maxDirectMemorySize=<size>
-			 *                and add -Dsun.nio.MaxDirectMemorySize=<size in bytes>
-			 */
-
-			UDATA value = 0;
-			char *optname = VMOPT_XXMAXDIRECTMEMORYSIZEEQUALS;
-			IDATA returnCode = convertMemorySizeToBytes(portLib, optString + strlen(VMOPT_XXMAXDIRECTMEMORYSIZEEQUALS), &value);
-			if ( OPTION_OK == returnCode ) {
-				const size_t bufferSize = strlen(SUN_NIO_MAXDIRECTMEMORYSIZE_EQUALS) + 16; /* allocate 16 byt bytes for maximum length decimal string and null */
```

Comment:
Should the definition of `SUN_NIO_MAXDIRECTMEMORYSIZE_EQUALS` be removed as well?

Reply:
Good idea.  Done.